browser/ui/webui/brave_rewards_page_ui.cc

```diff
@@ -35,13 +35,15 @@
 #include "content/public/browser/web_ui_message_handler.h"
 #include "content/public/common/bindings_policy.h"
 
-#if !defined(OS_ANDROID)
+#if defined(BRAVE_CHROMIUM_BUILD)
```

Comment:
Using BRAVE_CHROMIUM_BUILD here doesn't make sense because brave_rewards_page_ui.cc is only included for brave builds

Reply:
I explained that in DM, but will post here for the records. We need to distinguish between Android based on old repository and Android based on brave-core. Both of them have a `OS_ANDROID` define, but only brave-core build has `BRAVE_CHROMIUM_BUILD`. We can remove that `ifdef` completely when we fully migrate.

Reply:
ok, np